resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -72,6 +72,7 @@ start() {
 
     export HOME="$(getent passwd ${USER} | cut -d':' -f6)"
 
+    /usr/bin/install --directory --owner=$USER --group=$GROUP --mode=755 "/var/run/puppetlabs/${realname}"
```

Comment:
Again, probably inherited from the /etc/sysconfig|default script here.  Looks like there were other references to $USER at least before this change.